docs/api.md

```diff
@@ -121,4 +121,27 @@ Default: false
 
 Show output from DTrace or perf(1) tools.
 
-Default: false
\ No newline at end of file
+Default: false
+
+### `require('0x').ticksToTree(ticks) => Object`
+
+Build a stack tree from a list of stack samples (ticks). Useful for building
+custom visualizations. After a `collectOnly` run, the `ticks` data is stored in
+`{pid}.0x/ticks.json`. Read it using:
+
+```js
+const ticks = JSON.parse(fs.readFileSync('{pid}.0x/ticks.json', 'utf8'))
+```
+
+This function returns an object with properties:
+ - `merged` - a tree of stack frames, where each node includes both the
+   optimized and unoptimized calls for that frame.
+ - `unmerged` - a tree of stack frames, where each node refers to either the
+   optimized or unoptimized calls.
+
+Each node has properties:
```

Comment:
We add some extra properties on top of the existing ones from time to time (e.g., `isInit`. I am not sure if we actually need that property and I believe we do not but I did not yet test that properly and I do not want to remove it without having proper tests in place).

I suggest to add a comment that there might be some further properties as well that are not there by default.